### DISCUSSION TITLE
Kotlin library

### DISCUSSION THREAD

**Discussion by nullishamy** [catppuccin/catppuccin#1827] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Kotlin palette lib similar to existing language palettes

**Comment by d1snin** [comment]
The project is now ready for the review.

https://github.com/d1snin/catppuccin-kotlin

**Comment by sgoudham** [comment]
Tracked now at https://github.com/catppuccin/catppuccin/issues/1896
